<launch>
    <node pkg="nate_stretch_movement" type="crash_recovery_v2_node" name="Crash_Recovery_V2_Node" output="screen">
        <rosparam file="$(find nate_stretch_movement)/params/right_wall_hugger_params.yaml" command="load"/>
    </node>
</launch>